lib/Serverless.js

```diff
@@ -75,7 +75,8 @@ class Serverless {
       })
       .then(() => {
         // load all plugins
-        return this.pluginManager.loadAllPlugins(this.service.plugins);
+        this.pluginManager.loadAllPlugins(this.service.plugins);
+        return this.pluginManager.asyncPluginInit();
```

Comment:
I know it was like that before, but I would rather encapsulate that within `this.pluginManager.loadAllPlugins`, which seems totally safe to convert to _async_

Otherwise it looks as if we're leaking the internals of plugin manager outside